coverage/pom.xml

```diff
@@ -12,7 +12,7 @@
     <description>Compute aggregated test code coverage</description>
     <properties>
         <maven.deploy.skip>true</maven.deploy.skip>
-        <revision>1.0.0-SNAPSHOT</revision>
+        <revision>1.0.1-SNAPSHOT</revision>
```

Comment:
Should we create a CHANGELOG to list the changes in the version or should it be reserved only for releases?

Reply:
yes, we need to create github release but we need to have this PR merged first